src/Tribe/Assets.php

```diff
@@ -75,9 +75,9 @@ public function admin_enqueue_scripts() {
 					array( 'event-tickets-admin-css', 'tickets.css' ),
 					array( 'event-tickets-admin-refresh-css', 'tickets-refresh.css', array( 'event-tickets-admin-css' ) ),
 					array( 'event-tickets-admin-tables-css', 'tickets-tables.css', array( 'event-tickets-admin-css' ) ),
-					array( 'event-tickets-admin-js', 'tickets.js', array( 'jquery-ui-datepicker tribe-bumpdown' ) ),
```

Comment:
We need to have `tribe-bumpdown` as a dependency still. The `tribe-bumpdown` dependency was probably failing here because it was space separated rather than being a second element in the array.